rust/lance-encoding/src/encodings/physical/value.rs

```diff
@@ -348,4 +356,262 @@ pub(crate) mod tests {
             check_round_trip_encoding_random(field).await;
         }
     }
+
+    #[test_log::test(test)]
+    fn test_will_bitpack_allowed_types_when_possible() {
+        std::env::set_var("LANCE_USE_BITPACKING", "TRUE");
```

Comment:
There is an `EnvVarGuard` (or something like that) in one of the test utilities that will make this a bit easier.

Reply:
Yes this works, made this change.